diagnostic.md

```diff
@@ -23,7 +27,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$(".listerItem").parent.siblings("form")
```

Comment:
I believe you need to go up another level: `$('li').parent().parent().find('form')`